crypto-core/src/test/java/com/palantir/crypto2/io/CryptoStreamFactoryTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.crypto2.io;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import com.palantir.crypto2.cipher.AesCtrCipher;
+import com.palantir.crypto2.keys.KeyMaterial;
+import com.palantir.seekio.InMemorySeekableDataInput;
+import com.palantir.seekio.SeekableInput;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import org.junit.Test;
+
+public final class CryptoStreamFactoryTest {
+
+    @Test
+    public void testEncryptDecryptJce() throws IOException {
+        System.setProperty(CryptoStreamFactory.FORCE_JCE, "true");
```

Comment:
just stuck it at the end of the test since it should only apply to this specific test at the moment